src/state/selection/constants.ts

```diff
@@ -18,6 +18,8 @@ export const SET_AGENTS_VISIBLE = makeSelectConstant("set-agents-visible");
 export const RESET_AGENT_SELECTIONS_AND_HIGHLIGHTS = makeSelectConstant(
     "reset-selections-and-highlighted-agents"
 );
-export const SET_COLOR_CHANGES = makeSelectConstant("set-color-changes");
+export const APPLY_USER_COLOR_SELECTION = makeSelectConstant(
+    "apply-user-color-selection"
```

Comment:
nit, but since this is already the selection branch, you could call this `APPLY_USER_COLOR` (or `APPLY_USER_COLORS` whichever makes more sense)

Reply:
For sure, that's better